### DISCUSSION TITLE
LORA training on flux1-dev-bnb-nf4-v2?

### DISCUSSION THREAD

**Discussion by iqddd** [lllyasviel/stable-diffusion-webui-forge#1659] [Q&A]
Is that possible?
LORA trained on fp16 unet + fp16 T5XXL give poor generation results on bnb-nf4 (less character recognition, weaker style adherence, etc).
Web GUI from bmaltais/kohya_ss throws an error when starting training.

**Comment by iqddd** [comment]
Perhaps it is necessary to reverse conversion from NF4 to FP8/FP16?

**Comment by lllyasviel** [comment]
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1038

See also the section about lora precision

**Comment by iqddd** [comment]
Nothing about flux1-dev-bnb-nf4-v2-based training, just inference and related.
